components/CutoutPreviews.tsx

```diff
@@ -18,6 +18,8 @@ const FileMosaicComponent: React.FC<FileMosaicComponentProps> = ({ singleFile, m
     }
   }
 
+  const isMobile = useMediaQuery('(max-width: 768px)');
+
   const [imgSrc, setImgSrc] = useState<string | undefined>(undefined);
   const handleSee = (imageSource: string | undefined) => {
     setImgSrc(imageSource);
```

Comment:
**Suggestion:** Consider destructuring `singleFile` and `multipleFiles` at the beginning of the component to make the code cleaner and easier to read. [best practice]
```suggestion
    const FileMosaicComponent: React.FC<FileMosaicComponentProps> = (props) => {
      const { singleFile, multipleFiles } = props;
      const isMobile = useMediaQuery('(max-width: 768px)');
      const [imgSrc, setImgSrc] = useState<string | undefined>(undefined);
      const handleSee = (imageSource: string | undefined) => {
        setImgSrc(imageSource);
      }
```